packages/tasks/src/model-libraries.ts

```diff
@@ -508,6 +508,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	"magi-1": {
```

Comment:
as @Vaibhavs10 mentioned, you'll need to set `library_name: magi-1` in your model card metadata to make this work

Reply:
Thanks for your advice and I have set `library_name: "magi-1"` in the model card. Does the appearance of the symbol in the image mean the settings are correct?

![image](https://github.com/user-attachments/assets/08d92169-7f3b-4ea7-a9bc-3d198d4432be)

Reply:
yes!